clients/python/llmengine/data_types.py

```diff
@@ -163,17 +163,17 @@ class CreateLLMEndpointRequest(BaseModel):
     cpus: CpuSpecificationType
     gpus: int
     memory: StorageSpecificationType
-    gpu_type: GpuType
+    gpu_type: Optional[GpuType]
```

Comment:
just curious, why not default GpuType to `None` as well?

Reply:
The location in the code does specify a value for GpuType, and I think it's pretty reasonable to make us specify some value of GpuType. For most of the other ones I changed, there isn't a value passed in, so None seems like a reasonable default.